charts/steward/templates/clusterrole-tenant-controller.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-tenant-controller
+  labels:
+    {{- include "steward.labels" . | nindent 4 }}
+rules:
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["rolebindings"]
+  verbs: ["get","list","create"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles"]
+  verbs: ["bind"]
+  resourceNames:
+  {{- toYaml .Values.tenantController.mappableClusterRoles | nindent 2 }}
+- apiGroups: ["steward.sap.com"]
+  resources: ["tenants","tenants/status"]
+  verbs: ["get","list","patch","update","watch"]
+- apiGroups: [""]
+  resources: ["namespaces","resourcequotas","limitranges"]
+  verbs: ["create","delete","get","list","patch","update","watch"]
```

Comment:
The tenant controller doesn't need permissions for `resourcequotas` and `limitranges`, but the run controller!